05week/news-website-flexbox/index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+
+
+  <link rel="stylesheet" href="css/style.css">
+
+  <title>Bootstrap 101 Template Flex</title>
+
+
+  <link rel="stylesheet" href="css/style.css">
+
+</head>
+
+<body>
+
+  <h1>The Envato Tuts + Report</h1>
+  <h3><u>Read Tutorial</u></h3>
+  <br>
```

Comment:
Now that you know CSS and box-model avoid using `<br/>` to generate space. Use margin instead